### ISSUE TITLE
overridePermissions clipboard-read not working on createIncognitoBrowserContext()

### ISSUE THREAD

**Issue by CyberT33N** [puppeteer/puppeteer#6354]
The following code is viable for reading the clipboard in headless/headfull:
```javascript
var context = await client.defaultBrowserContext();
await context.overridePermissions('http://localhost', ['clipboard-read']);
page = await browser.newPage();
await page.goto( 'http://localhost/test/', {waitUntil: 'load', timeout: 35000});
// click button for clipboard..
let clipboard = await page.evaluate(`(async () => await navigator.clipboard.readText())()`);
```

But when you later start incognito its not working anymore:
```javascript
const incognito = await client.createIncognitoBrowserContext();
page = await incognito.newPage();
```

and you get:
- DOMException: Read permission denied.

I currently try to figure out to combine both.. Anybody know how to set overridePermissions inside of the new incognito window?

Please notice I do not want to use the incognito chrome arg at the start. I want to manually create new incognito pages inside of my scripts with correct overridePermissions.

**Comment by AndrewPahomovOtto** [comment]
Hello from 2022 ✋  Still not work

**Comment by maikeg-alves** [comment]
The issue is still not resolved, Puppeteer v20.7.2

**Comment by unhud** [comment]
Still not solved haha
